script.js

```
@@ -0,0 +1,542 @@
+            return [...products].sort((a, b) => a.price - b.price);
+        case 'price-high':
+            return [...products].sort((a, b) => b.price - a.price);
+        case 'rating':
+            return [...products].sort((a, b) => b.rating - a.rating);
+        default:
+            return products;
+    }
+}
+
+// Handle sorting change
+function sortProducts() {
```

Biome (2.4.4)

[error] Shouldn't redeclare 'sortProducts'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)